opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListCreation.tsx

```diff
@@ -133,24 +154,35 @@ const ExclusionListCreationForm: FunctionComponent<ExclusionListCreationFormProp
             renderOption={(
               props: React.HTMLAttributes<HTMLLIElement>,
               option: Option,
-            ) => <li key={option.value} {...props}>{option.label}</li>}
-            textfieldprops={{ label: t_i18n('Entity types') }}
+            ) => <li key={option.value} {...props}>
+              <ItemIcon type={option.type} />
+              <span style={{ padding: '0 4px 0 4px' }}>{option.label}</span>
+            </li>}
+            textfieldprops={{ label: t_i18n('Apply on indicator observable types') }}
+            required
           />
-          <CustomFileUploader setFieldValue={setFieldValue} />
-          <Field
-            component={AutocompleteField}
-            name="action"
-            fullWidth={true}
-            style={fieldSpacingContainerStyle}
-            options={actions}
-            renderOption={(
-              props: React.HTMLAttributes<HTMLLIElement>,
-              option: string,
-            ) => <li key={option} {...props}>{option}</li>}
-            textfieldprops={{ label: t_i18n('Action') }}
-            disabled
-          />
-
+          <div style={fieldSpacingContainerStyle}>
+            <span onClick={toggleFile} style={{ cursor: 'pointer' }}>{t_i18n('Content')}</span>
+            <Switch
+              defaultChecked
+              checked={isFileChecked}
+              onChange={toggleFile}
+            />
+            <span onClick={toggleFile} style={{ cursor: 'pointer' }}>{t_i18n('File')}</span>
+          </div>
```

Comment:
I have interrogations about this pattern of using a toggle button + a tricky span to simulate a second option. I don't think we have this in other places in the app and seems a bit odd when using to see the toggle grey as disabled when selecting `content`.

IMO should use already existing fields we have like a select.

Reply:
Updated :

![image](https://github.com/user-attachments/assets/84b583cb-fe63-476d-a252-d157fae0fb35)

I hope it's more clearly ?

Reply:
validate by Romain